setup.py

```diff
@@ -29,8 +27,6 @@
           'Topic :: Multimedia :: Graphics :: 3D Rendering',
           'Topic :: Scientific/Engineering :: Visualization',
     ],
-    install_requires=['jupyter', 'vpython'],
-    #cmdclass=cmdclass('vpnotebook/data'),
-    #package_data={'vpython': ['data/*'], 'vpnotebook': ['data/kernel.json']},
+    install_requires=['jupyter', 'vpnotebook'],
```

Comment:
Oy! It's good that you caught that. Let me see whether I can deal with
this, as I'd like to put this all together today.

Bruce

On Mon, Jan 25, 2016 at 6:52 AM, Steve Spicklemire <notifications@github.com
> wrote:

> In setup.py
> <https://github.com/BruceSherwood/vpython-jupyter/pull/2#discussion_r50695044>
> :
>
> > @@ -29,8 +27,6 @@
> >            'Topic :: Multimedia :: Graphics :: 3D Rendering',
> >            'Topic :: Scientific/Engineering :: Visualization',
> >      ],
> > -    install_requires=['jupyter', 'vpython'],
> > -    #cmdclass=cmdclass('vpnotebook/data'),
> > -    #package_data={'vpython': ['data/*'], 'vpnotebook': ['data/kernel.json']},
> > +    install_requires=['jupyter', 'vpnotebook'],
>
> Oops. It's gone from vpython.py, but it's still in rate_control.py ;-(.
> Sorry, I missed that.
>
> Bruce, should I work on removing numpy from rate_control? Is that
> something you intend to have done by someone?
>
> thx.
> -steve
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BruceSherwood/vpython-jupyter/pull/2/files#r50695044>.
>